src/main/src/callback.rs

```diff
@@ -1,7 +1,7 @@
 use core::nonzero::NonZero;
 use process;
 
-#[derive(Clone,Copy)]
+#[derive(Clone,Copy,Eq,PartialEq)]
```

Comment:
Not sure about this one. I use it to check if the application calling the functions is the one that "owns" the device, but is this identifier supposed to be stable over calls? (Works on my machine™, in any case.)